Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/ModalInfoView/ModalInfoViewModel.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 /// for other code such as making an API call.
 public protocol ModalInfoViewModel {
     var title: GDSLocalisedString { get }
-    var body: GDSLocalisedString { get }
+    var body: NSAttributedString { get }
```

Comment:
I wonder if we should look at adding to `GDSLocalisedString` so that it can handle `NSAttributedString` as well?

Reply:
tech debt ticket [here](https://govukverify.atlassian.net/browse/DCMAW-7096)

Reply:
My feeling is adding this to `GDSLocalisedString` should be the approach to doing this piece of work rather than treating it as tech debt.

I think we should be cautious about changing the APIs of public protocols such as this, especially if we think we might ultimately change it back to a `GDSLocalisedString` later. 

Right now, there are probably very few consumers of this, but in time, they might be numerous and the impact will not all be known to us, so changing the protocol should be done only if we really want / need to.

Reply:
Think it is almost there now. But if we are expecting an imminent piece of work to revert a breaking change to the public API we should probably not be making the change in the first place 🙂